spec/gibbon/gibbon_spec.rb

```diff
@@ -251,34 +258,52 @@
 
     it "not throw exception if the Export API replies with a JSON hash containing a key called 'error'" do
       @gibbon.throws_exceptions = false
-      allow(Gibbon::Export).to receive(:post).and_return(Struct.new(:body).new(MultiJson.dump({'error' => 'bad things'})))
+      reply = MultiJson.dump({:error => 'bad things'})
+      stub_request(:post, @url + 'sayHello/').
+        to_return(:body => reply, :status => 200)
 
       @gibbon.say_hello(@body)
     end
 
     it "throw exception if configured to and the Export API replies with a JSON hash containing a key called 'error'" do
       @gibbon.throws_exceptions = true
-      params = {:body => @body, :timeout => 30}
-      reply = Struct.new(:body).new MultiJson.dump({'error' => 'bad things', 'code' => '123'})
-      allow(Gibbon::Export).to receive(:post).and_return reply
+      reply = MultiJson.dump({:error => 'bad things', :code => '123'})
+      stub_request(:post, @url + 'sayHello/').
+        to_return(:body => reply, :status => 200)
 
       expect {@gibbon.say_hello(@body)}.to raise_error(Gibbon::MailChimpError)
     end
 
     it "should handle a single empty space response without throwing an exception" do
       @gibbon.throws_exceptions = true
-      allow(Gibbon::Export).to receive(:post).and_return(Struct.new(:body).new(" "))
+      stub_request(:post, @url + 'sayHello/').
+        to_return(:body => " ", :status => 200)
+      #allow(Gibbon::Export).to receive(:post).and_return(Struct.new(:body).new(" "))
 
-      expect(@gibbon.say_hello(@body)).to eq([])
+      expect(@gibbon.say_hello(@body)).to eq([[]])
```

Comment:
What's up with this change? Why would an "empty" response return an array of 1 empty array? See #102 and #113